pkgs/electrs/generate.sh

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p git gnupg
+set -e
+
+echo "Creating temporary directory"
+mkdir /tmp/generate_electrs_expression
+cd /tmp/generate_electrs_expression
+git clone https://github.com/romanz/electrs
+
+echo "Checking out latest release"
+cd /tmp/generate_electrs_expression/electrs
+latesttag=$(git describe --tags `git rev-list --tags --max-count=1`)
+git checkout ${latesttag}
+echo "Latest release is ${latesttag}"
+
+echo "Getting Roman Zeyde's PGP key"
+gpg --recv-keys 15C8C3574AE4F1E25F3F35C587CAE5FA46917CBB
```

Comment:
always better if you change state to ask the user if he/she really wants that. Something like `Do you want to import Roman Zeyde's PGP key (15C...) [yn]`

Reply:
Added in 3227a0151f6559e6a2e5bd30e4736d7396c0399c